### DISCUSSION TITLE
options.ksfile should be read from inst.ks boot param

### DISCUSSION THREAD

**Discussion by T0MASD** [rhinstaller/anaconda#4205] [Ideas]
I was wondering if there's a way for anaconda to set `options.ksfile` value from `inst.ks` boot param?
At the moment, when machine is booted using:
```
linux ($root)/images/pxeboot/vmlinuz root=live:LABEL=live ro rd.live.image inst.ks=file:///install.cfg systemd.unit=anaconda.target
```
 `options.ksfile` defaults to `/run/install/ks.cfg`  instead of expected `/run/initramfs/live/install.ks`  as seen from:

![anaconda](https://i.imgur.com/55gEtoD.png)

**Comment by T0MASD** [comment]
For now I'm using the following workaround:
```
find /usr -path "*/site-packages/pyanaconda/argument_parsing.py" -print0 | xargs -0 sed -i 's/\/run\/install\/ks.cfg/\/run\/initramfs\/live\/install.ks/g' 
```
I've raised a bug https://bugzilla.redhat.com/show_bug.cgi?id=2101097

**Comment by poncovka** [comment]
Hello, I don't think it is possible to change the default location at this moment. It should be downloaded there by stage1 during the boot of the `boot.iso`. Could you please provide more info about the image that you are trying to boot? It doesn't look like a supported setup.
